kwasm-lemmas.md

```diff
@@ -324,17 +366,12 @@ module MEMORY-CONCRETE-TYPE-LEMMAS
 ```
 
 ```k
-    rule #getRange(BM, START, WIDTH) => 0
-      requires notBool (WIDTH >Int 0)
-      [simplification]
-    rule #getRange(BM, START, WIDTH) => #get(BM, START) +Int (#getRange(BM, START +Int 1, WIDTH -Int 1) *Int 256)
-      requires          WIDTH >Int 0
-       andBool #isByteMap(BM)
-      [simplification]
-
-    rule #wrap(WIDTH, N) => N modInt (1 <<Int WIDTH)
+    rule #wrap(N, #getRange(BM, ADDR, WIDTH)) => #getRange(BM, ADDR, WIDTH -Int (N /Int 8))
+      requires N modInt 8 ==Int 0
       [simplification]
```

Comment:
`N >=Int 0`

Reply:
Also just realized this lemma bad, it should be `N /Int 8`, and only when the `WIDTH` is larger. Will fix and ensure it still passes.